tk-demos/widget

```diff
@@ -1,5 +1,6 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/env perl
 
+use warnings;
```

Comment:
is it really necessary?
probably '-w' above is the solution?
or even BEGIN{$^W=1}
?
for my taste, warnings.pm became considerably worse in recent perls